setup.py

```diff
@@ -47,7 +47,7 @@ def check_env_flag(name: str, default: bool = False) -> bool:
 
 # define install requirements
 install_requires_list = ['packaging', 'numpy', 'schema', 'pyyaml']
-opt_install_requires_list = ['neural_compressor', 'transformers==4.34.1']
+opt_install_requires_list = ['neural_compressor', 'transformers']
```

Comment:
To be discussed...

Reply:
@PenghuiCheng Request it to be `<=4.34.1` for pruning and sparse features. Will be upgraded soon.